examples/stamp/stamp.bal

```diff
@@ -21,47 +21,48 @@ type Foo record {
 
 public function main() {
 
-    // This stamps the anydata type array to an int array
+    // Stamp an `anydata` array as an `int` array.
     anydata[] anydataArray = [1, 2, 3, 4];
     int[]|error intArray = int[].stamp(anydataArray);
     if (intArray is int[]) {
-        io:println("Element type of the array is Int");
+        io:println("Element type of the array is 'int'");
     }
 
-    // This stamps the int type array to json
+    // Stamp an `int` array as `json`.
     int[] intArrayValue = [1, 2, 3, 4];
     json|error jsonValue = json.stamp(intArrayValue);
     if (jsonValue is json[]) {
-        io:println("Int array is stamped as JSON array");
+        io:println("Element type of the array is 'json'");
     }
 
-    // This stamps the Teacher type variable as the Employee since fields
-    // are matching.
+    // Stamp a `Teacher` record as an `Employee` record.
+    // This would not return an `error` since all fields match.
     Teacher t1 = { name: "Raja", age: 25, status: "single",
                             batch: "LK2014", school: "Hindu College" };
     Employee e = Employee.stamp(t1);
     string school = <string> e.school;
     io:println("School of the Employee is " + school);
 
-    // This stamps the map type variable as the record type
+    // Stamp an `anydata` map as a `Teacher` record.
+    // This could return an `error` since all required fields may not be present.
     map<anydata> anydataMap = { name: "Raja", age: 25, status: "single",
                             batch: "LK2014", school: "Hindu College" };
     Teacher|error teacherValue = Teacher.stamp(anydataMap);
     if (teacherValue is Teacher) {
-        io:println("Map is stamped as Teacher record type");
+        io:println("'map<anydata>' is stamped as a 'Teacher'");
     }
 
-    // This stamps the tuple type variable as the int array
-    (int,int) intTuple = (1, 2);
+    // Stamp a tuple as an `int` array
```

Comment:
Wouldn't tuple be within back ticks as well?

Reply:
Didn't introduce back ticks since tuple is not a keyword. Similar to array.